ddsrouter_core/src/cpp/communication/rpc/RPCBridge.cpp

```diff
@@ -365,6 +373,9 @@ void RPCBridge::transmit_(
                     continue;
                 }
 
+                // Perform write + add entry to registry atomically -> avoid reply processed before entry added to registry
```

Comment:
_line 363_

Probably this should be a continue, or a tsnh.
Break will block this bridge forever

Reply:
I don't see how break would block the bridge, it would get out of the while loop I believe.

However, I changed this to and if-else clause so `release_payload` is called regardless of the condition.